taichi/ir/statements.h

```diff
@@ -362,11 +421,14 @@ class LoopUniqueStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE
 };
 
+/**
+ * A global load.
```

Comment:
Could you add a few more details, explaining that we are using this to access not only the SNodes, but also external array, TLS, BLS?